servers/rendering/rendering_device.cpp

```diff
@@ -449,6 +450,7 @@ void RenderingDevice::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("compute_list_set_push_constant", "compute_list", "buffer", "size_bytes"), &RenderingDevice::_compute_list_set_push_constant);
 	ClassDB::bind_method(D_METHOD("compute_list_bind_uniform_set", "compute_list", "uniform_set", "set_index"), &RenderingDevice::compute_list_bind_uniform_set);
 	ClassDB::bind_method(D_METHOD("compute_list_dispatch", "compute_list", "x_groups", "y_groups", "z_groups"), &RenderingDevice::compute_list_dispatch);
+	ClassDB::bind_method(D_METHOD("compute_list_dispatch_indirect", "compute_list", "buffer", "offset"), &RenderingDevice::compute_list_dispatch_indirect);
```

Comment:
Looks like you missed implementing this

Reply:
Godot actually already has [implemented indirect compute shader dispatch](https://github.com/godotengine/godot/blob/32cde9f7949b79ee6cf06596e712444b5ded2a32/drivers/vulkan/rendering_device_vulkan.cpp#L8515). Hence the PR is "exposing" indirect CSD rather than adding its functionality as with indirect drawing.

I simply added the bind method for exposure to Godot scripting for that case.